docs/core/compatibility/toc.yml

```diff
@@ -1,42 +1,32 @@
-- name: Application compatibility
+- name: Evaluate breaking changes
+  href: index.md
+- name: Categories
+  href: categories.md
+- name: Page selectors
```

Comment:
This TOC node "page selectors" goes to an H1 ".NET Core breaking changes" which is a jarring disconnect and the H1 doesn't really match the page content. Maybe make the H1 something like "Page selectors for .NET Core breaking changes"?

That said, the page duplicates navigation provided by the TOC, and SkyRay shows less than 400 page views for it - I would consider deleting it.

Reply:
I changed the H1 to "Breaking change selectors".